doc/classes/ARVRServer.xml

```diff
@@ -121,6 +121,7 @@
 	</methods>
 	<members>
 		<member name="world_scale" type="float" setter="set_world_scale" getter="get_world_scale">
+			The world scale, see the ARVROrigin documentation for more information.
```

Comment:
Don't just repeat the member name as the description. See above scale description.

Is there a link to the ARVROrigin documentation?

Reply:
i should really read more of what i commit, sorry :/